pyccl/power.py

```diff
@@ -175,7 +175,13 @@ def sigma8(cosmo, p_of_k_a=None):
     Returns:
         float: RMS variance in top-hat sphere of radius 8 Mpc/h.
     """
-    return sigmaR(cosmo, 8.0 / cosmo['h'], p_of_k_a=p_of_k_a)
+    psp = parse_pk2d(cosmo, p_of_k_a, is_linear=True)
+    status = 0
+    s8, status = lib.sigma8(cosmo.cosmo, psp, status)
+    check(status, cosmo)
```

Comment:
This is slightly faster than calling `sigmaR`, because we already had a function named `lib.sigma8` at the C-level.

Reply:
I'm not sure this is any faster at all. Since you're in the business of simplifying things in this PR: why don't we get rid of the sigma8 function in C altogether?

Reply:
I measure it to be marginally faster, but maybe you're right, the C-layer is meant to do the heavy lifting, not to define our convenience functions. I removed it from the C-layer and replaced python's sigma8 with the old version.

Reply:
While `sigma8` in C is a convenience function, it is used internally, and removing it would actually complicate the codebase, instead of making it simpler, so it's not a good idea to remove it.
I measure the proposed version of the python `sigma8` to be marginally faster than calling python's `sigmaR`, so maybe it's best if we actually make use of `lib.sigma8` since it's already there (at the expense of making the code 3 lines longer).